rethinkdb/rethinkdb.d.ts

```diff
@@ -36,10 +36,15 @@ declare module "rethinkdb" {
     export class Cursor {
         hasNext(): boolean;
         each(cb: (err: Error, row: any) => void, done?: () => void): void;
+        each<T>(cb: (err: Error, row: T) => void, done?: () => void): void;
```

Comment:
@vladima I'm not completely clear about your question.

My intent is to allow support for typed `each` method in addition to untyped, eg:

```
cursor.each<IMyRowType>((err: Error, row: IMyRowType) => ...);
```

Is this a wrong way to approach this?